packages/api-client/src/auth.ts

```diff
@@ -15,15 +15,15 @@ export default class Auth {
     });
   }
 
-  login({ email, password }: PostLoginRequest): Promise<void> {
-    return this.client.post({
```

Comment:
Does this really make any difference? You just don't want to return anything?

Reply:
You have a lot of changes of this type in this PR. Is it a general cleanup to remove unused return values?

Reply:
It's related to [this comment](https://github.com/arrocke/gloss-translation/pull/176#discussion_r1313956947). Before post requests were returning essentially an empty body, but now they have the location header. For these post requests, the location header isn't relevant so I'm preventing it from being returned from the function.